examples/example1.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE tokens (
```

Comment:
let's name the files so they make sense from the outside, fx. this one could be called `inline_primary_key.sql`